misc/e2e-aws.sh

```diff
@@ -226,7 +226,18 @@ EOF
     cd ${WORKDIR}
 }
 
+stage-debug() {
+  # Output logs for failed units
+  exec_on master1 "sudo systemctl list-units --failed | \
+          cut -d ' ' -f2 | \
+          tail -n+2 | \
+          head -n -7 | \
+          xargs -i sh -c 'echo logs for {} ; sudo journalctl --no-pager -u {}; echo'"
+}
+
 stage-destroy() {
+  stage-debug
```

Comment:
`stage-debug || true` 

we also need to survive failure if any, otherwise it won't run `terraform destroy`